docs/docs/manual/cellediting.md

```diff
@@ -93,7 +93,7 @@ OpenRefine will do a number of cleanup operations behind the scenes in order to
 
 You can start the process in two ways: using the dropdown menu on your column, select <span class="menuItems">Edit cells</span> → <span class="menuItems">Cluster and edit…</span>; or create a text facet and then press the “Cluster” button that appears in the facet box. 
 
-![A screenshot of the Clustering window.](/img/cluster.png)
+![A screenshot of the Clustering window.](/img/cluster.jpg)
```

Comment:
I think it is generally preferable to use the `png` format rather than `jpg` for images on the web.
See also my previous comment about cropping the screenshot tightly around the border of the dialog.